src/index.html

```diff
@@ -22,6 +22,393 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__wrap">
+        <a href="/">
+          <img
+            class="header__logo"
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
```

Comment:
Add classes everywhere
```suggestion
            <li class="nav__item">
```